cl/search/constants.py

```diff
@@ -307,6 +306,30 @@
     Opinion.TRIAL_COURT: "trial-court-document",
 }
 
+recap_document_indexed_fields = [
+    "id",
+    "docket_entry_id",
+    "description",
+    "entry_number",
+    "entry_date_filed",
+    "short_description",
+    "document_type",
+    "document_number",
+    "pacer_doc_id",
+    "plain_text",
+    "attachment_number",
+    "is_available",
+    "page_count",
+    "cites",
+]
+
+recap_document_filters = [
+    "available_only",
+    "description",
+    "document_number",
+    "attachment_number",
+]
```

Comment:
These arrays are unused. They're likely remnants of an older approach. We can remove them.

Reply:
yes, I used these in a earlier approach that changed later. We can safely remove them.